plonky3/src/prover/mod.rs

```diff
@@ -86,24 +86,50 @@ mod tests {
 
     /// Prove and verify execution
     fn run_test_goldilocks(pil: &str) {
+        run_test_goldilocks_publics(pil, None)
+    }
+
+    fn run_test_goldilocks_publics(pil: &str, malicious_publics: Option<Vec<GoldilocksField>>) {
         let mut pipeline = Pipeline::<GoldilocksField>::default().from_pil_string(pil.to_string());
 
         let pil = pipeline.compute_optimized_pil().unwrap();
         let witness_callback = pipeline.witgen_callback().unwrap();
         let witness = pipeline.compute_witness().unwrap();
 
-        let proof = Plonky3Prover::new(&pil).prove(&witness, witness_callback);
+        let prover = Plonky3Prover::new(&pil);
+        let proof = prover.prove(&witness, witness_callback);
 
         assert!(proof.is_ok());
+
+        if let Some(publics) = malicious_publics {
+            prover.verify(&proof.unwrap(), &[publics]).expect("")
```

Comment:
```suggestion
            prover.verify(&proof.unwrap(), &[publics]).unwrap()
```
Any advantage of using `expect("")`?

Reply:
I used `expect("")` to propagate the error message from calling `verify`, but not sure if the same error message appears every time verification fails